Marlin/language_de.h

```diff
@@ -203,6 +205,9 @@
 #define MSG_INFO_MIN_TEMP                   "Min Temp"
 #define MSG_INFO_MAX_TEMP                   "Max Temp"
 #define MSG_INFO_PSU                        "Stromversorgung"
+#define MSG_DRIVE_STRENGTH                  "Motorkraft"
```

Comment:
Your right, but i dont see another method to drive the printer without using a motor